dll/win32/shell32/wine/shellord.c

```diff
@@ -1388,8 +1388,15 @@ HRESULT WINAPI SHWinHelp(HWND hwnd, LPCWSTR pszHelp, UINT uCommand, ULONG_PTR dw
  */
 BOOL WINAPI SHRunControlPanel (LPCWSTR commandLine, HWND parent)
 {
+#ifdef __REACTOS__
+    HINSTANCE hr;
+    TRACE("SHRunControlPanel(%s, %p)n", debugstr_w(commandLine), parent);
+    hr = ShellExecuteW(parent, NULL, commandLine, NULL, NULL, SW_NORMAL);
+    return ((int)hr > 32);
```

Comment:
What about 'As of Windows Vista, this function always returns FALSE'? Especially as ROS Shell is compiled as NT6.

Reply:
Any suggestion ?

Reply:
At least a reminder TODO comment?

Moreover, while implementing `SHRunControlPanel` NT5.2- code seems fine, do we actually want to use this very function everywhere, as it would "break" as NT6+?

Reply:
yes, sure !